Gems/LyShine/Code/Source/Tests/internal/test_UiTextComponent.cpp

```diff
@@ -3522,6 +3522,7 @@ void UiTextComponent::UnitTestLocalization(CLyShine* lyshine, IConsoleCmdArgs* /
             }
         }
     }
+    AZ_Assert(initLocSuccess, "Test failed");
```

Comment:
Since this is just for a UnitTest, I suggest just removing the `initLocSuccess` variable